src/uct/cuda/gdr_copy/gdr_copy_iface.c

```diff
@@ -27,6 +26,19 @@ static ucs_config_field_t uct_gdr_copy_iface_config_table[] = {
      ucs_offsetof(uct_gdr_copy_iface_config_t, super),
      UCS_CONFIG_TYPE_TABLE(uct_iface_config_table)},
 
+    {"BW", "0MBs,get_dedicated:250MBs,put_shared:6911MBs",
```

Comment:
why default is 0?

Reply:
Default is 0 only for values that are not mentioned later. That's how current key-value parser works. I mean that according to the master branch code, `get_shared` and `put_dedicated` are 0, so it is the shortest way to define them to that value.